CONTRIBUTING.md

```diff
@@ -86,6 +86,17 @@ Makefile allow you to install needed tools:
 $ make install.tools
 ```
 
+### Prerequisite before build
```

Comment:
Could you change line 49 please?  I know it's pre-existing, but it points to install.md which then points to podman.io.  Can you just change line 49 from:
```
Read the [install documentation to see how to install dependencies](install.md) .
```
to

```
Read the [install documentation to see how to install dependencies](https://podman.io/getting-started/installation#build-and-run-dependencies).
```
(Note, I removed an extra space before the ending period).  Then I think we'll cover @luap99's suggestion and @rhatdan's concern.  I personally like having this short crib sheet as you have it here for most experienced contributors, then the pointer to the deeper dive for the less experienced ones.  Touch that up and I think this is gtg.
TYVM!

Reply:
Done, thanks for your help.